components/collective-page/sections/TopFinancialContributors.js

```diff
@@ -0,0 +1,66 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import memoizeOne from 'memoize-one';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+
+import { getTopContributors } from '../../../lib/collective.lib';
+import { CollectiveType } from '../../../lib/constants/collectives';
+
+import Container from '../../Container';
+import { H4 } from '../../Text';
+import ContainerSectionContent from '../ContainerSectionContent';
+import TopContributors from '../TopContributors';
+
+const TopContributorsContainer = styled.div`
+  padding: 32px 16px;
+  margin-top: 48px;
+  background-color: #f5f7fa;
+`;
+
+/**
+ * Top financial contributors widget.
+ */
+const SectionTopFinancialContributors = ({ collective, financialContributors }) => {
+  const isEvent = collective.type === CollectiveType.EVENT;
+  const getTopContributorsMemoized = memoizeOne(getTopContributors);
```

Comment:
This memoization has no effect.

Since the component only has two props and is well contained, the simplest is to rather export it with `React.memo`. It will not re-render unless the `financialContributors` or `collective` change.